cmd/minikube/cmd/start_test.go

```diff
@@ -126,6 +127,71 @@ func TestStartClusterUpWithFlag(t *testing.T) {
 	assertCommandLineArguments(expectedArguments, t)
 }
 
+func TestShellProxyVariableWithProxyFlag(t *testing.T) {
+	setUp(t)
+	defer os.RemoveAll(testDir)
+	defer minitesting.ResetDefaultRoundTripper()
+	defer SetRunner(util.RealRunner{})
+	defer viper.Reset()
+
+	expectedShellProxyVariable := "http_proxy=http://localhost:3128 https_proxy=https://localhost:3128 no_proxy=localhost,127.0.0.1"
+
+	viper.Set("http-proxy", "http://localhost:3128")
```

Comment:
@hferentschik we can set both values also something like if same server serve http and https request then it's dynamically chosen by application.

```
export {http,https}_proxy="http://proxy-server:port"
```